### ISSUE TITLE
get/set-Worksheet function

### ISSUE THREAD

**Issue by fpbarthel** [awalker89/openxlsx#69]
Hi,

I'm looking for a function to get and set an existing worksheet in a workbook object. Eg.

wb  = loadWorkbook("test.xlsx")
ws = getWorksheet(wb, sheet=1, header=F) ## function that operates like read.xlsx with input as wb object instead of filename

## manipulate the data frame eg.
ws[,1] = ws[,1] * 100

wb = setWorksheet(wb, sheet=1, data=ws)

## Save it
saveWorkbook(wb, "text.v2.xlsx")

Thanks if this could be implemented!

Floris

**Comment by sda030** [comment]
Yes, would be very useful when batch reading multiple sheets and joining these together.
